content/blog/ggplot2-3-5-0/index.Rmd

```diff
@@ -0,0 +1,385 @@
+Also, it has gained a `staplewidth` argument that can be used to draw staples: horizontal lines at the end of the boxplot whiskers.
+
+```{r boxplot_outliers_staples}
+#| fig.alt = "Boxplot showing the price of diamonds per cut. The y-axis does
+#|  not go much beyond the whiskers, and whiskers are decorated with a staple."
+ggplot(diamonds, aes(cut, price)) +
+  geom_boxplot(outliers = FALSE, staplewidth = 0.5)
+```
+
+The scales functions now do a better job at reporting *which* scale has encountered an error.
+
+```{r scale_messages, error=TRUE}
+#| fig.alt= "Scatterplot showing body weight minus one versus brain weight of 
+#|  mammals. The x-axis is log-transformed."
+scale_colour_brewer(breaks = 1:5, labels = 1:4)
+
+ggplot(mpg, aes(class, displ)) +
+  geom_boxplot() +
+  scale_x_continuous()
+
+ggplot(msleep, aes(bodywt - 1, brainwt)) +
+  geom_point(na.rm = TRUE) +
+  scale_x_log10()
+```
+
+## Acknowledgements
+
+Thank you to all people who have contributed issues, code and comments to this release:
+
+[&#x0040;ashgreat](https://github.com/ashgreat), [&#x0040;banbh](https://github.com/banbh), [&#x0040;benimwolfspelz](https://github.com/benimwolfspelz), [&#x0040;ccsarapas](https://github.com/ccsarapas), [&#x0040;danli349](https://github.com/danli349), [&#x0040;davidhodge931](https://github.com/davidhodge931), [&#x0040;dieghernan](https://github.com/dieghernan), [&#x0040;edent](https://github.com/edent), [&#x0040;eliocamp](https://github.com/eliocamp), [&#x0040;f2il-kieranmace](https://github.com/f2il-kieranmace), [&#x0040;garyzhubc](https://github.com/garyzhubc), [&#x0040;Generalized](https://github.com/Generalized), [&#x0040;giadasp](https://github.com/giadasp), [&#x0040;jan-glx](https://github.com/jan-glx), [&#x0040;jimjam-slam](https://github.com/jimjam-slam), [&#x0040;jtlandis](https://github.com/jtlandis), [&#x0040;klin333](https://github.com/klin333), [&#x0040;krlmlr](https://github.com/krlmlr), [&#x0040;manjumc1975](https://github.com/manjumc1975), [&#x0040;math-mcshane](https://github.com/math-mcshane), [&#x0040;matthewjnield](https://github.com/matthewjnield), [&#x0040;olivroy](https://github.com/olivroy), [&#x0040;paulatn240](https://github.com/paulatn240), [&#x0040;retodomax](https://github.com/retodomax), [&#x0040;Rong-Zh](https://github.com/Rong-Zh), [&#x0040;steveharoz](https://github.com/steveharoz), [&#x0040;tbates](https://github.com/tbates), [&#x0040;teunbrand](https://github.com/teunbrand), [&#x0040;thomasp85](https://github.com/thomasp85), [&#x0040;tjebo](https://github.com/tjebo), [&#x0040;warnes](https://github.com/warnes), [&#x0040;wbvguo](https://github.com/wbvguo), [&#x0040;willgearty](https://github.com/willgearty), [&#x0040;Yunuuuu](https://github.com/Yunuuuu), and [&#x0040;yuw444](https://github.com/yuw444).
```

Comment:
This seems short. Are you sure it's accurate?

Reply:
I forgot there were some hotfix releases that branched off the previous release while development on main continued, so it didn't include the folks who contributed in the meanwhile. Included contributors from 3.4.1 onwards now.